lib/+ne7/+mat/getParams.m

```diff
@@ -0,0 +1,37 @@
+function params = getParams(params,vararginput)
+
+% function params = getParams(params,vararginput)
+%
+% Gives the ability to pass the param as a structure to a function
+%
+% 2009-02-24 MF
+
+
+if ~isempty(vararginput)
+
+    char = zeros(1,length(vararginput));
```

Comment:
Since both `char` and `struc` are built-in Matlab functions, would you mind changing them to something else?